examples/cpp/Makefile

```diff
@@ -1,17 +1,23 @@
 MOLE = ../../src/cpp
 
-CXX ?= g++
+#ARMA = /Users/aneeshsrinivas/Documents/GitHub/armadillo-14.0.3
+#if you are running on MACOS, uncomment all of the lines which have a # for mac os. 
+#CXX ?= clang++ #for mac os
+CXX ?= g++ #for non mac users.
 
 ifeq ($(DEBUG),1)
 CXXFLAGS = -O0 -g
 else
 CXXFLAGS = -O3
 endif
-CXXFLAGS += -fopenmp -DARMA_DONT_USE_WRAPPER
+CXXFLAGS += -fopenmp -DARMA_DONT_USE_WRAPPER #for non mac users.
+#CXXFLAGS +=  -DARMA_DONT_USE_WRAPPER #for mac os
```

Comment:
Remove the comments on these lines as well. Some macOS users might have openmp installed.